src/annotations.jl

```diff
@@ -38,16 +33,22 @@ a presumed `onda.annotation` table. Returns `annotations`.
 
 This function will throw an error in any of the following cases:
 
-- `Legolas.validate(annotations, Legolas.Schema("onda.annotation@1"))` throws an error
-- `Annotation(row)` errors for any `row` in `Tables.rows(annotations)`
+- `Legolas.validate(Tables.schema(annotations), AnnotationV1SchemaVersion())` throws an error
+- `AnnotationV1(r)` errors for any `r` in `Tables.rows(annotations)`
 - `annotations` contains rows with duplicate `id`s
 """
-validate_annotations(annotations) = _fully_validate_legolas_table(annotations, Legolas.Schema("onda.annotation@1"), :id)
+validate_annotations(annotations) = _fully_validate_legolas_table(:validate_annotations, annotations, AnnotationV1, AnnotationV1SchemaVersion(), :id)
 
 #####
-##### utilities
+##### `merge_overlapping_annotations`
 #####
 
+@schema "onda.merged-annotation" MergedAnnotation
+
+@version MergedAnnotationV1 > AnnotationV1 begin
```

Comment:
Hmm. It could be. I'm not worried about this code needing to change much any time soon, so I'll go ahead/document export it in case folks want to extend it downstream